release-notes/release-notes52.md

```diff
@@ -116,10 +116,14 @@ For information on using premium skins and icon packs, see: [Premium Skins and I
 
 The {{site.productname}} 5.2 release includes **Spellchecker Pro** 2.0.
 
-**Spellchecker Pro** 2.0 includes:
+**Spellchecker Pro** 2.0:
```

Comment:
I'm not sure if it's worth mentioning, but 2.0.0 also has a new internal engine that's faster at parsing the document.

Reply:
fixed